Dockerfile

```diff
@@ -2,10 +2,12 @@ FROM golang:1.16-buster as go
 ENV GO111MODULE=on
 ENV CGO_ENABLED=0
 ENV GOBIN=/bin
-RUN go get github.com/go-delve/delve/cmd/dlv@v1.5.0
-RUN go get github.com/edwarnicke/dl
-RUN dl https://github.com/spiffe/spire/releases/download/v0.11.1/spire-0.11.1-linux-x86_64-glibc.tar.gz | \
-    tar -xzvf - -C /bin --strip=3 ./spire-0.11.1/bin/spire-server ./spire-0.11.1/bin/spire-agent
+RUN go get github.com/go-delve/delve/cmd/dlv@v1.6.0
+ADD https://github.com/spiffe/spire/releases/download/v0.11.1/spire-0.11.1-linux-x86_64-glibc.tar.gz .
+RUN tar xzvf spire-0.11.1-linux-x86_64-glibc.tar.gz -C /bin --strip=3 ./spire-0.11.1/bin/spire-server ./spire-0.11.1/bin/spire-agent
```

Comment:
@edwarnicke Do you mind if we get rid of using dl?

Reply:
Nope